lib/rules/jsx-handler-names.js

```diff
@@ -68,6 +77,16 @@ module.exports = {
       return prefix === false;
     }
 
+    function isInlineHandler(node) {
+      return node.value.expression.type === 'ArrowFunctionExpression';
+    }
+
+    function isLocalInlineHandler(node) {
```

Comment:
I'm not exactly sure what this is returning, but it's not a boolean. Should this be called 'getInlineHandler' or something like that?

Reply:
This function is the part to be examined for local variable if inline handler.

The AST in the case of inline handler and local variable is as follows. (node.value.expression.body.callee.name)
![Screen Shot 2020-08-25 at 2 23 00 AM](https://user-images.githubusercontent.com/16266103/91076073-e8d62100-e679-11ea-9267-edbc8c642b97.png)

If the inline handler is not a local variable, the AST is as follows. (node.value.expression.body.callee.property.name)
![Screen Shot 2020-08-25 at 2 25 23 AM](https://user-images.githubusercontent.com/16266103/91076301-410d2300-e67a-11ea-9b1f-a7ecd56f3a15.png)

In this case, `getPropValue` is a better name because it's not a boolean and it returning propValue. Would you agree with it?

Reply:
Sounds good!

Reply:
I remove getHandlerName function, because below code is more flexible.

`context.getSourceCode().getText(node.value.expression.body.callee).replace(/^this\.|.*::/, '')`

https://github.com/yannickcr/eslint-plugin-react/commit/ddcf12cf4b1e75b32b84807b3fc06d3a2c573d14#diff-ccf82a24ec6f364517e4d1f4aed7a0a8R114